immut/list/list.mbt

```diff
@@ -29,6 +29,18 @@ pub fn T::to_string[A : Show](xs : T[A]) -> String {
   Show::to_string(xs)
 }
 
+pub impl[A : ToJson] ToJson for T[A] with to_json(self) {
```

Comment:
FromJson is also needed

Reply:
@Yoorkin Can I implement this in another PR?

Reply:
@Yoorkin Unfortunately, I've found a bug in `@immut/sorted_set.T::op_equal()` that has broken my quicktests. I'll fix that first.

Reply:
@Yoorkin All fixed.